Userland/Libraries/LibM/math.cpp

```diff
@@ -56,57 +57,6 @@ enum class RoundingMode {
     ToEven = FE_TONEAREST
 };
 
-template<typename T>
```

Comment:
Can you put this code move in a separate commit, explaining why it's necessary? (standalone use of FloatExtractor)